tf-build/mongo/action.yml

```diff
@@ -0,0 +1,15 @@
+name: "CDP Mongo User Creation"
+description: "Triggers update of mongo user roles"
+
+inputs:
+  topicArn:
+    description: Which sns topic to send the message to
+  environment: 
+    description: Environment the queue is in
+
+runs:
+  using: "composite"
+  steps:
+    - name: Update Mongodb Users
+      run: |
+        aws sns publish --topic-arn ${{ inputs.topicArn }} --message "$(cat environments/${{ inputs.environment }}/resources/tenant_services.json)"
```

Comment:
the message can be the path of the file `--message file://environments/${{ inputs.environment }}/resources/tenant_services.json` - assuming that variable will be replaced